docs/settings.rst

```diff
@@ -785,6 +785,21 @@ the server will return NODATA for A/AAAA queries for such names.
   In PowerDNS Authoritative Server 4.0.x, this setting did not exist and
   ALIAS was always expanded.
 
+.. _setting-resolve-across-zones:
+
+``resolve-across-zones``
```

Comment:
please add a versionadded thing with 5.0.0

(if we decide to backport, we can update it)

Reply:
done!